tarantool_test.go

```diff
@@ -721,6 +721,44 @@ func BenchmarkSQLSerial(b *testing.B) {
 	}
 }
 
+func TestFutMultipleGetGetTyped(t *testing.T) {
+	conn := test_helpers.ConnectWithValidation(t, server, opts)
+	defer conn.Close()
+
+	fut := conn.Call17Async("simple_concat", []interface{}{"1"})
+
+	for i := 0; i < 30; i++ {
```

Comment:
So we are able to fetch result multiple times from the future? Is it really expected?

Reply:
> So we are able to fetch result multiple times from the future? There are no any similar API in sync requests since they return the response after execution?

A user can call Future.Get() multiple times before the patch. So for sync API nothing change.

Reply:
> I think we should check the same for errors, just in case

A good idea! I've added such tests.